lightning/src/ln/channel.rs

```diff
@@ -1445,58 +1448,58 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
-		self.their_max_htlc_value_in_flight_msat = cmp::min(msg.max_htlc_value_in_flight_msat, self.channel_value_satoshis * 1000);
-		self.local_channel_reserve_satoshis = msg.channel_reserve_satoshis;
-		self.their_htlc_minimum_msat = msg.htlc_minimum_msat;
-		self.their_to_self_delay = msg.to_self_delay;
-		self.their_max_accepted_htlcs = msg.max_accepted_htlcs;
+		self.counterparty_dust_limit_satoshis = msg.dust_limit_satoshis;
+		self.counterparty_max_htlc_value_in_flight_msat = cmp::min(msg.max_htlc_value_in_flight_msat, self.channel_value_satoshis * 1000);
+		self.counterparty_selected_channel_reserve_satoshis = msg.channel_reserve_satoshis;
+		self.counterparty_htlc_minimum_msat = msg.htlc_minimum_msat;
+		self.counterparty_selected_contest_delay = msg.to_self_delay;
+		self.counterparty_max_accepted_htlcs = msg.max_accepted_htlcs;
 		self.minimum_depth = msg.minimum_depth;
 
-		let their_pubkeys = ChannelPublicKeys {
+		let counterparty_pubkeys = ChannelPublicKeys {
 			funding_pubkey: msg.funding_pubkey,
 			revocation_basepoint: msg.revocation_basepoint,
 			payment_point: msg.payment_point,
 			delayed_payment_basepoint: msg.delayed_payment_basepoint,
 			htlc_basepoint: msg.htlc_basepoint
 		};
 
-		self.local_keys.on_accept(&their_pubkeys, msg.to_self_delay, self.our_to_self_delay);
-		self.their_pubkeys = Some(their_pubkeys);
+		self.holder_keys.on_accept(&counterparty_pubkeys, msg.to_self_delay, self.holder_selected_contest_delay);
+		self.counterparty_pubkeys = Some(counterparty_pubkeys);
 
-		self.their_cur_commitment_point = Some(msg.first_per_commitment_point);
-		self.their_shutdown_scriptpubkey = their_shutdown_scriptpubkey;
+		self.counterparty_cur_commitment_point = Some(msg.first_per_commitment_point);
+		self.counterparty_shutdown_scriptpubkey = counterparty_shutdown_scriptpubkey;
 
 		self.channel_state = ChannelState::OurInitSent as u32 | ChannelState::TheirInitSent as u32;
 
 		Ok(())
 	}
 
-	fn funding_created_signature<L: Deref>(&mut self, sig: &Signature, logger: &L) -> Result<(Transaction, LocalCommitmentTransaction, Signature), ChannelError> where L::Target: Logger {
+	fn funding_created_signature<L: Deref>(&mut self, sig: &Signature, logger: &L) -> Result<(Transaction, HolderCommitmentTransaction, Signature), ChannelError> where L::Target: Logger {
 		let funding_script = self.get_funding_redeemscript();
 
-		let local_keys = self.build_local_transaction_keys(self.cur_local_commitment_transaction_number)?;
-		let local_initial_commitment_tx = self.build_commitment_transaction(self.cur_local_commitment_transaction_number, &local_keys, true, false, self.feerate_per_kw, logger).0;
-		let local_sighash = hash_to_message!(&bip143::SighashComponents::new(&local_initial_commitment_tx).sighash_all(&local_initial_commitment_tx.input[0], &funding_script, self.channel_value_satoshis)[..]);
+		let keys = self.build_local_transaction_keys(self.cur_holder_commitment_transaction_number)?;
```

Comment:
Note that there are a ton of places in channel.rs that still have the /local_// conversion instead of /local_/holder_/. It seems they're all in function-scope variables so I'm ok with it if you dont want to fix it, but figured I'd note in case you or Jeff feel strongly.

Reply:
Yes at first try I was aiming to savagely converse everything then figured out a lot of references are meddled with ChannelState names.

I'm aiming to fix it when we refactor/cleanup further Channel internals. And have some on anchor branch.